pkg/utils/driver.go

```diff
@@ -55,6 +57,11 @@ func BindDpdkDriver(pciAddr, driver string) error {
 	err = ioutil.WriteFile(bindPath, []byte(pciAddr), os.ModeAppend)
 	if err != nil {
 		glog.Errorf("BindDpdkDriver(): fail to bind driver for device %s: %s", pciAddr, err)
+		_, err := os.Readlink(filepath.Join(sysBusPciDevices, pciAddr, "iommu_group"))
```

Comment:
when driver bind fails, there might be two reasons:
1) iommu not enabled in BIOS.  (will this result in error when setting iommu kernel argument ?)
2) iommu is enabled in BIOS, iommu kernel argument is not configured.

Shall we distinguish the two with different error messages?

Reply:
Isn't 2. covered by the operator itself?

Reply:
There was an issue reported from customer-0 that vfio bind failed but VT-d is enabled in BIOS. We didn't find the root cause as there is no clear reproducer. The fix propsed for aforementioned issue is to at: https://github.com/openshift/sriov-network-operator/pull/179. But I don't know if there are other causes.

Reply:
Understood, thanks

Reply:
After #179, the daemon will always reboot the node if the IOMMU is not enabled in Kernel. So if case 2, the daemon will reboot the host before trying to load the driver.